core/state_transition.go

```diff
@@ -331,6 +333,11 @@ func (st *StateTransition) preCheck(gasBailout bool) error {
 // However if any consensus issue encountered, return the error directly with
 // nil evm execution result.
 func (st *StateTransition) TransitionDb(refunds bool, gasBailout bool) (*ExecutionResult, error) {
+	input1 := st.state.GetBalance(st.msg.From())
```

Comment:
Wrap this into section only executable for bor (so we do not need to waste time on this during Ethereum execution) Also perhaps create a bool variables inside `StateTransition` indicating `Parlia` and `for`, to check `st.isBor` instead of checking `st.evm.ChainConfig().Bor != nil`